src/lib.rs

```diff
@@ -1658,6 +1701,74 @@ impl Instance {
                             },
                         )))
                     }
+                    GeneratedTrampoline::Utf8CopyTranscoder { from, to } => {
+                        let from = from.as_u32();
+                        let to = to.as_u32();
+                        let ty = ty.with_name(format!("transcode-copy-utf8-{}-{}", from, to));
+                        Ok(Extern::Func(wasm_runtime_layer::Func::new(
+                            ctx.as_context_mut().inner,
+                            ty,
+                            move |_ctx, args, results| {
+                                println!("transcode-copy-utf8-{from}-{to}({args:?}, {results:?})");
+                                bail!("transcode-copy-utf8 is not implemented")
+                            },
+                        )))
+                    }
+                    GeneratedTrampoline::AlwaysTrap => {
+                        let ty = ty.with_name("always-trap");
+                        Ok(Extern::Func(wasm_runtime_layer::Func::new(
+                            ctx.as_context_mut().inner,
+                            ty,
+                            move |_ctx, args, results| {
+                                println!("always-trap({args:?}, {results:?})");
+                                Err(wasmtime_environ::Trap::AlwaysTrapAdapter.into())
+                            },
+                        )))
+                    }
+                    GeneratedTrampoline::ResourceTransferOwn => {
```

Comment:
I'll have to investigate their functionality further and properly implement these trampolines, but this is sufficient to at least instantiate my component and run some functions